### DISCUSSION TITLE
Lab meeting presentation on GitHub for ecologists

### DISCUSSION THREAD

**Discussion by Aariq** [SORTEE-Github-Hackathon/manuscript#110] [Show and tell]
Hi all, at the last meeting I mentioned I gave a talk for lab meeting inspired by y'all.  Here's a link to the [repo](https://github.com/Aariq/github-for-ecologists) and [slides](https://aariq.github.io/github-for-ecologists/slides/slides.html#1).  Feel free to fork and edit and adapt!

**Comment by wavingtowaves** [comment]
Thanks so much for sharing Eric!

On Fri, Feb 25, 2022 at 11:12 AM Eric R Scott ***@***.***>
wrote:

> Hi all, at the last meeting I mentioned I gave a talk for lab meeting
> inspired by y'all. Here's a link to the repo
> <https://github.com/Aariq/github-for-ecologists> and slides
> <https://aariq.github.io/github-for-ecologists/slides/slides.html#1>.
> Feel free to fork and edit and adapt!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SORTEE-Github-Hackathon/manuscript/discussions/110>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTX7SK3UW56D5EEAXFZ7E3U47IC5ANCNFSM5PLBZ5HA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Rob Crystal-Ornelas, PhD
Postdoctoral Scholar
Lawrence Berkeley National Lab | ESS-DIVE
Pronouns: he/him/his
